app/models/user.rb

```diff
@@ -1,12 +1,15 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Let's make sure that the application asks for confirmation of email.
 `:confirmable`

Reply:
Hello @Team. By mistake I approved your project wich has a change to be performed. Kindly fix the change requested above before merging this pull request. you need to add `:confirmable` in your user model.

your code will look as bellow:
````ruby
devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
````

after correcting this, you can merge your pull request and proceed with the next project. no need to submit for another review on this pull request.

Thank you.

Reply:
Thank you  very much @uwadonat  . I made the changes